csaf_2.0/json_schema/csaf_json_schema.json

```diff
@@ -951,6 +958,8 @@
         "minProperties": 1,
         "properties": {
           "acknowledgments": {
+            "title": "Vulnerability acknowledgments",
+            "description": "Contains a list of acknowledgment elements associated with this vulnerability item.",
```

Comment:
@sthagen Should we use `associated with the current vulnerability item.` instead?

Reply:
Good question. The entry in question is part of a vulnerability object being an item in the array of all vulnerabilities of the advisory / CSAF document. The term `this` Is even used in some funny programming languages as reference to such an entity, others use `self` Per convention but allow any valid identifier if the programmer insists. I would keep the `this` As the `current` may provide to many notions like an `iterator like concept` Or even a `temporal aspect` Both of which we do not want to convey.

Reply:
👍🏽